internal/cli/config/config.go

```diff
@@ -67,13 +74,47 @@ func (opts *configOpts) Save() error {
 }
 
 func (opts *configOpts) Run() error {
-	helpLink := "https://docs.atlas.mongodb.com/configure-api-access/"
+	fmt.Printf(`You are setting up %s
+
+All values are optional and you can use environment variables (MCLI_*) instead.
+
+Enter [?] on any option to get help.
+
+`, config.ToolName)
+
+	q := opts.accessQuestions()
+	if err := survey.Ask(q, opts); err != nil {
+		return err
+	}
+
+	q = opts.defaultQuestions()
+	if err := survey.Ask(q, opts); err != nil {
+		return err
+	}
+
+	if err := opts.Save(); err != nil {
+		return err
+	}
+
+	fmt.Printf("\n%s is all set now\n", config.ToolName)
+	fmt.Printf("You can use [%s config set] to change more settings. \n", config.ToolName)
+	if config.Name() != config.DefaultProfile {
+		fmt.Printf("To use this profile remember to set the flag [-%s %s]\n", flag.ProfileShort, config.Name())
+	}
+	return nil
+}
```

Comment:
```suggestion
	if config.Name() != config.DefaultProfile {
		fmt.Printf("To use this profile, you must set the flag [-%s %s] for every command.\n", flag.ProfileShort, config.Name())
	}
	return nil
}
```

Also I think we should move this up under the first sentence, after line 99. So that the order makes a little more sense:
Your profile is now configured.
(To use this profile, you must set the flag [-%s %s] for every command.)
You can use [%s config set] to change these settings at a later time.